cicecore/cicedynB/dynamics/ice_dyn_evp_1d.F90

```diff
@@ -1186,51 +1185,49 @@ subroutine evp_copyin_v2(nx,ny,nblk,nx_glob,ny_glob,
-!v1    call gather_global_ext(G_cxp, I_cxp, master_task, distrb_info)
-!v1    call gather_global_ext(G_cym, I_cym, master_task, distrb_info)
-!v1    call gather_global_ext(G_cxm, I_cxm, master_task, distrb_info)
-!v1    call gather_global_ext(G_tinyarea, I_tinyarea, master_task, distrb_info)
-!v1    call gather_global_ext(G_waterx, I_waterx, master_task, distrb_info)
-!v1    call gather_global_ext(G_watery, I_watery, master_task, distrb_info)
-    call gather_global_ext(G_cdn_ocn, I_cdn_ocn, master_task, distrb_info)
-    call gather_global_ext(G_aiu, I_aiu, master_task, distrb_info)
-    call gather_global_ext(G_uocn, I_uocn, master_task, distrb_info)
-    call gather_global_ext(G_vocn, I_vocn, master_task, distrb_info)
-    call gather_global_ext(G_forcex, I_forcex, master_task, distrb_info)
-    call gather_global_ext(G_forcey, I_forcey, master_task, distrb_info)
-    call gather_global_ext(G_Tbu, I_Tbu, master_task, distrb_info)
-    call gather_global_ext(G_umassdti, I_umassdti, master_task, distrb_info)
-    call gather_global_ext(G_fm, I_fm, master_task, distrb_info)
-    call gather_global_ext(G_uarear, I_uarear, master_task, distrb_info)
-    call gather_global_ext(G_tarear, I_tarear, master_task, distrb_info)
-    call gather_global_ext(G_strintx, I_strintx, master_task, distrb_info)
-    call gather_global_ext(G_strinty, I_strinty, master_task, distrb_info)
-    call gather_global_ext(G_uvel_init, I_uvel_init, master_task, distrb_info)
-    call gather_global_ext(G_vvel_init, I_vvel_init, master_task, distrb_info)
-    call gather_global_ext(G_strength, I_strength, master_task, distrb_info)
-    call gather_global_ext(G_uvel, I_uvel, master_task, distrb_info)
-    call gather_global_ext(G_vvel, I_vvel, master_task, distrb_info)
-    call gather_global_ext(G_dxt, I_dxt, master_task, distrb_info)
-    call gather_global_ext(G_dyt, I_dyt, master_task, distrb_info)
-    call gather_global_ext(G_stressp_1, I_stressp_1, master_task, distrb_info)
-    call gather_global_ext(G_stressp_2, I_stressp_2, master_task, distrb_info)
-    call gather_global_ext(G_stressp_3, I_stressp_3, master_task, distrb_info)
-    call gather_global_ext(G_stressp_4, I_stressp_4, master_task, distrb_info)
-    call gather_global_ext(G_stressm_1, I_stressm_1, master_task, distrb_info)
-    call gather_global_ext(G_stressm_2, I_stressm_2, master_task, distrb_info)
-    call gather_global_ext(G_stressm_3, I_stressm_3, master_task, distrb_info)
-    call gather_global_ext(G_stressm_4, I_stressm_4, master_task, distrb_info)
-    call gather_global_ext(G_stress12_1, I_stress12_1, master_task, distrb_info)
-    call gather_global_ext(G_stress12_2, I_stress12_2, master_task, distrb_info)
-    call gather_global_ext(G_stress12_3, I_stress12_3, master_task, distrb_info)
-    call gather_global_ext(G_stress12_4, I_stress12_4, master_task, distrb_info)
+    call gather_global_ext(G_icetmask,   I_icetmask,   master_task, distrb_info, 0      )
```

Comment:
It's curious that icetmask has 0 at the end of the argument list and iceumask has .false.!  I see that is how they are defined.  No need to fix it here...